rocAL_pybind/examples/rocAL_training_example/README.md

```diff
@@ -0,0 +1,113 @@
+# ImageNet training in PyTorch
+
+This implements training of popular model architectures, such as ResNet, AlexNet, and VGG on the ImageNet dataset.
+This version has been modified to use rocAL. It assumes that the dataset is raw JPEGs from the ImageNet dataset. If offers CPU and GPU based pipeline for rocAL - use rocal-cpu switch to enable CPU one and use rocal-gpu switch to enable GPU one.
+
+To run use the following command
+```bash
+rm *.pth.tar # Remove older checkpoints saved in the folder if the example has been run before
+python3 main.py -a resnet50 --dist-url='tcp://127.0.0.1:4321' --dist-backend 'nccl' --multiprocessing-distributed --world-size 1 --rank 0 -j$(nproc) --batch-size 1024 --rocal-cpu --epochs 91 /media/imageNetCompleteDataset/
+```
+
+## Requirements
```

Comment:
requirements should come before the the above instructions to run

Reply:
Moved the rocAL training command after Requirements section